app/models/accounting/quickbooks/connection.rb

```diff
@@ -50,6 +50,17 @@ def realm_id
         data[:realm_id] unless data.blank?
       end
 
+      def last_updated_at
+        DateTime.parse(data[:last_updated_at]) unless data.blank? || data[:last_updated_at].blank?
```

Comment:
How are we handling time zones here? This app is globally in Eastern time right now but that will undoubtedly change. I'd like to standardize on using `Time.zone.parse` in most places. We should also ensure that the date that gets stored in the JSON below includes a timezone (we can convert them all to UTC if you like and include `UTC`, or just write out the local timezone, but one or the other needs to be there). So perhaps explicitly doing the string conversion instead of relying on the default one?